app/definitions/rest/v1/index.ts

```
@@ -21,6 +21,7 @@
 import { type DirectoryEndpoint } from './directory';
 import { type AutoTranslateEndpoints } from './autotranslate';
 import { type ModerationEndpoints } from './moderation';
+import { QrCodeScannerEndpoints } from './QrScanner';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)